lib/database_consistency/checkers/index_checkers/index_checker.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module DatabaseConsistency
+  module Checkers
+    # The base class for table checkers
+    class IndexChecker < BaseChecker
+      attr_reader :model, :index
+
+      def initialize(model, index)
+        @model = model
+        @index = index
+      end
+
+      def column_or_attribute_name
+        @column_or_attribute_name ||= index.name.to_s
```

Comment:
It's not really the column nor attribute name, I can put something else if you prefer

Reply:
You did right 👍 I hope I'll do a refactor to just name it as `key`.